src/components/header.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
```

Comment:
I see you import this but don't use it. Is this for the future?

Reply:
Yeah I was trying to use it in order to access i18n from the I18nextProvider. I need to look into it more!